src/wp-includes/class-wp-theme-json.php

```diff
@@ -1934,6 +1934,16 @@ protected static function compute_style_properties( $styles, $settings = array()
 			return $declarations;
 		}
 
+		$can_use_cached = ! wp_is_development_mode( 'theme' );
+
+		$args      = func_get_args(); // phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.Changed
+		$cache_key = 'compute_style_properties_' . md5( wp_json_encode( $args ) );
+		$cache     = wp_cache_get( $cache_key, 'wp-styles' );
```

Comment:
This code is still being called if the theme is in development mode. Could you please review the approach taken in [this section](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/global-styles-and-settings.php#L464-L485) of the code and consider applying a similar approach here?